### ISSUE TITLE
To see latest changes every time I've to run "sassdoc [source] [dest]". How can I automate it like watch functionality?

### ISSUE THREAD

**Issue by ksharifbd** [SassDoc/sassdoc#424]

**Comment by pascalduez** [comment]
Hi, 

opening several issues won't bring you faster replies.
Keep in mind your talking to humans not robots.

SassDoc doesn't ship with a files watching functionality.
You would need to integrate it with a task runner like [Grunt](http://sassdoc.com/grunt/), [Gulp](http://sassdoc.com/gulp/), [Broccoli](http://sassdoc.com/broccoli/), ... to achieve this behaviour.

**Comment by ksharifbd** [comment]
Hello, 

Thank you for the reply and sorry for the inconvenience. I opened another issue by mistake forgetting that I've already opened same issue just days ago. I wanted to delete it but could not.

I use gulp and the sassdoc site says "Starting from version 2.0 and higher SassDoc is able to integrate directly in your Gulp pipelines without any plugin. In such the gulp-sassdoc plugin has been deprecated." I was confused. Now understood the issue.
